bitnami/airflow/values.yaml

```
@@ -1953,7 +1953,12 @@ redis:
     password: ""
     existingSecret: ""
   architecture: standalone
+  ## @param redis.master.service.ports.redis Redis&reg; port
+  ##
   master:
+    service:
```

I'd move the comment one line below

```suggestion
  master:
    ## @param redis.master.service.ports.redis Redis&reg; port
    ##
    service:
```